app/code/Magento/StoreGraphQl/Model/Resolver/Store/StoreConfigDataProvider.php

```diff
@@ -83,14 +107,13 @@ private function getBaseConfigData(StoreInterface $store) : array
             'weight_unit' => $storeConfig->getWeightUnit(),
             'base_url' => $storeConfig->getBaseUrl(),
             'base_link_url' => $storeConfig->getBaseLinkUrl(),
-            'base_static_url' => $storeConfig->getSecureBaseStaticUrl(),
+            'base_static_url' => $storeConfig->getBaseStaticUrl(),
```

Comment:
what is the diff between secure and not secure? https?
I think we have a Magento setting for this even..bu if we have https,..we should provide https